src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -642,6 +647,135 @@ public async Task TestFilterOnInnerNestedFields()
             await ExecuteAndValidateResult(_graphQLQueryName, gqlQuery, dbQuery);
         }
 
+        #region Field Level Auth
+        /// <summary>
+        /// Tests that the field level query filter succeeds requests when filter fileds are authorized
+        /// </summary>
+        [TestMethod]
+        public async Task TestQueryFilterFieldAuth_AuthorizedField()
+        {
+            string gqlQuery = @"{
+                earths(first: 1, " + QueryBuilder.FILTER_FIELD_NAME + @" : {id : {eq : """ + _idList[0] + @"""}})
+                { 
+                    items {
+                        id
+                    }
+                }
+            }";
+
+            string dbQuery = $"SELECT top 1 c.id FROM c where c.id = \"{_idList[0]}\"";
+            await ExecuteAndValidateResult("earths", gqlQuery, dbQuery);
+        }
+
+        /// <summary>
+        /// Tests that the field level query filter fails requests when filter fileds are unauthorized
```

Comment:
```suggestion
        /// Tests that the field level query filter fails authorization when filter fields are
        /// unauthorized because the field 'name' on object type 'earth' is an excluded field of the read
        /// operation permissions defined for the anonymous role.
```

Reply:
Updated